link-grammar/Makefile.am

```diff
@@ -205,7 +205,7 @@ EXTRA_DIST=                         \
 
 # Clean up generated file that Windows MSVC compilation leaves behind.
 clean-local:
-	rm -f $(top_builddir)/link-grammar/link-features.h
+	rm -f $(top_srcdir)/link-grammar/link-features.h
```

Comment:
Oof. Nothing should ever get written into the srcdir during build. This suggests a different bug.

Reply:
This is Windows, not Un*x...
The LG README documented stuff used to tell to copy `link-grammar/link-features.h.in` into link-`grammar/link-features.h`. Now the Windows build stuff does the same (but better - w/changing the @variables@).

This  cleanup command is if you later want to compile on a Un*x machine (my src dir is remote-mounted on Windows, with each branch at its own directory).

*Proposal*:
I can instead change the build procedure to create it in the `msvc14` directory, and add this directory to the Windows LG-build include path. The only problem is that this name `msvc14` is a moving target... so when renamed, a change will need to be done in several places too (but maybe I will find a way to not use an explicit name in the definitions).

Reply:
Or we can just have this cleanup rule to support compiling from Windows in the same directory....

Reply:
I'm not sure I care, at this point. In principle source dirs should be read-only during builds, but there are so many other packages that break this - maybe the majority of all packages ... there's only so many decisions one can make in a day, I don't know if I want to make this one.

Reply:
Possibilities:
1. Remove the current rule (it is harmful, as it removes a file created by `configure`).
2. Change it to remove from srcdir, which is a no-op for proper Un*x usage (but can be a bad surprise for those who anyway use the srcdir for build).
3. Also (1), plus putting it under `msvc14` (or whatever it will be renamed to).

Just tell me what to do...

Reply:
I will remove the rule altogether, and will later (on conversion to MSVC15) will try to see if I can generate `link-grammar/link-features.h` under `msvcX` w/o mentioning `msvcX` anywhere.